charts/velocity-registry/values.yaml

```diff
@@ -0,0 +1,39 @@
+replicaCount: 1
+
+image:
+  repository: registry
+  tag: 2.8.1
+  pullPolicy: IfNotPresent
+
+env:
+  RegistryAuth:
+    name: REGISTRY_AUTH
+    value: htpasswd
+  RegistryAuthRealm:
+    name: REGISTRY_AUTH_HTPASSWD_REALM
+    value: Registry Realm
+  RegistryAuthPath:
+    name: REGISTRY_AUTH_HTPASSWD_PATH
+    value: /auth/htpasswd
+
+registryContainerPort: 5000
+registryServicePort: 5000
+
+RegistryResources:
+  limits:
+    cpu: "1"
+    memory: 1Gi
+  requests:
+    cpu: 100m
+    memory: 250Mi
+
+secrets:
+  registryAuthName: registry-auth
+  registryConfigName: registry-docker-config
+  DefaultUsername: root
+  DefaultPassword: asdf1234!@#$
+
+volume:
+  pvcName: velocity-registry-pvc
+  storage: 10Gi
+  storageClass: ebs-sc
```

Comment:
Notice this assumes running on AWS. 
Check to see what other helm charts are doing in order not to be vendor specific here

Reply:
I don't think there is a way to not write something here. its for the customer to change if he runs something else. i'll take it with Product \ @orensham  to verify.

Reply:
I agree with @yarelm, but for now, we decided to support only AWS with the product. 
Anyway, we want to be generic next step for the day after Monday, so let's create a task to allow us to change it according to the customer input.

Reply:
please open a task and comment here (don't include the task info as this is a public repo)